addon/components/bs-form/element/control/checkbox.js

```diff
@@ -0,0 +1,35 @@
+import Control from '../control';
+
+/**
+
+ @class FormElementControlCheckbox
+ @namespace Components
+ @extends Components.FormElementControl
+ @private
+ */
+export default Control.extend({
+  attributeBindings: [
```

Comment:
This could get ugly with the global attributes, as well. Same issue we discussed with `nav.anchor`. I think we need to live with it in this case. I considered enhancing `attributeBindings` to allow a hash usage that would allow attribute passthrough, but decided not to go there. But my investigations led me to write the missing documentation for it.